### DISCUSSION TITLE
How should info.plist be managed?

### DISCUSSION THREAD

**Discussion by ryan-ju** [ionic-team/capacitor#6720] [Q&A]
I don't know if this is a common problem that many people face, but I can't find any good docs on this.

Basically, what's the recommended way to manage native configs like info.plist?

I have tried editing it directly, but `cap sync` just replaces it.

What's the recommended approach to add permissions to info.plist, without having to edit it manually every time after a `cap sync`?  Does capacitor have a config option for this?

**Comment by jcesarmobile** [comment]
Capacitor never updates your native project files, well, except for the `npx cap migrate` command as it exists for making changes in the native project required in the new Capacitor version the command is migrating to.

Maybe you are updating a plugin `Info.plist` and not your app's `Info.plist`?
